docs/next/guides/building-and-testing/building.md

```diff
@@ -9,83 +9,263 @@ tags:
+  - During release, these files are published into the npm repository.
+
+`npm run build:umd`
+  - Creates the following bundles compatible with the Universal Module Definition:
+    - `dist/handsontable.js`
+    - `dist/handsontable.css`
+    - `dist/handsontable.full.js`
+    - `dist/handsontable.full.css`
+
+`npm run build:umd.min`
+  - Creates the minified bundles compatible with the Universal Module Definition:
+    - `dist/handsontable.min.js`
+    - `dist/handsontable.min.css`
+    - `dist/handsontable.min.full.js`
+    - `dist/handsontable.min.full.css`
+
+`npm run build:walkontable`
+  - PLACEHOLDER
+
+`npm run build:languages`
+  - Creates the [language](@/guides/internationalization/internationalization-i18n.md) bundles compatible with the Universal Module Definition, for example:
+    - `dist/languages/de-DE.js`
+    - `dist/languages/all.js`
+
+`build:languages.es`
+  - PLACEHOLDER
+
+`npm run build:languages.min`
+   - Creates the minified [language](@/guides/internationalization/internationalization-i18n.md) bundles compatible with the Universal Module Definition, for example:
+     - `dist/languages/de-DE.min.js`
+     - `dist/languages/all.min.js`
+:::
+
+### Building the Angular package
+
+To build the Angular package:
+1. Make sure you meet the [building requirements](#building-requirements).
+3. Go to `./wrappers/angular`.
+4. Run `npm run build`.<br>Only the Angular package builds.
+
+To build the Angular package from the root directory:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to the root directory.
+3. Run `npm run in angular build`.<br>Only the Angular package builds.
+
+#### Angular build tasks
+
+You can also run individual Angular `build` tasks:
+
+::: details Angular build tasks
+`prebuild`
+  - PLACEHOLDER
+
+`build`
+  - PLACEHOLDER
+
+`postbuild`
+  - PLACEHOLDER
+:::
+
+### Building the React package
+
+To build the React package:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to `./wrappers/react`.
+3. Run `npm run build`.<br>Only the React package builds.
+
+To build the React package from the root directory:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to the root directory.
+3. Run `npm run in react build`.<br>Only the React package builds.
+
+#### React build tasks
+
+You can also run individual React `build` tasks:
+
+::: details React build tasks
+`build:commonjs`
+  - Transpiles files into the `CommonJS` format.
+  - During release, these files are published into the npm repository.
+
+`build:umd`
+  - Creates the following bundles compatible with the Universal Module Definition:
+    - `dist/handsontable.js`
+    - `dist/handsontable.css`
+    - `dist/handsontable.full.js`
+    - `dist/handsontable.full.css`
+
+`build:es`
+  - Transpiles files into the `import/export` format.
+  - During release, these files are published into the npm repository.
```

Comment:
```suggestion
    - `wrappers/react/es/react-handsontable.js`
```

(side note: I'm not sure if we should use relative or absolute-ish paths here? With `wrappers/react` or without?)

Reply:
Unless there are some factors I'm not aware of, I'd go for fullest paths possible: 7bd7512